azext_edge/edge/_help.py

```diff
@@ -780,6 +780,167 @@ def load_iotops_help():
+    """
+
+    helps[
+        "iot ops dataflow endpoint create local-mqtt"
+    ] = """
+        type: command
+        short-summary: Create or replace a Azure IoT Operations Local MQTT data flow endpoint.
+    """
+
+    helps[
+        "iot ops dataflow endpoint create localstorage"
+    ] = """
+        type: command
+        short-summary: Create or replace a local storage data flow endpoint.
+    """
+
+    helps[
+        "iot ops dataflow endpoint update"
+    ] = """
+        type: group
+        short-summary: Update the properties of an existing dataflow endpoint resource.
+    """
+
+    helps[
+        "iot ops dataflow endpoint update adls"
+    ] = """
+        type: command
+        short-summary: Update the properties of an existing dataflow endpoint resource for Azure Data Lake Storage Gen2.
+    """
+
+    helps[
+        "iot ops dataflow endpoint update adx"
+    ] = """
+        type: command
+        short-summary: Update the properties of an existing dataflow endpoint resource for Azure Data Explorer.
+    """
+
+    helps[
+        "iot ops dataflow endpoint update custom-kafka"
+    ] = """
+        type: command
+        short-summary: Update the properties of an existing dataflow endpoint resource for custom kafka broker.
+    """
+
+    helps[
+        "iot ops dataflow endpoint update custom-mqtt"
+    ] = """
+        type: command
+        short-summary: Update the properties of an existing dataflow endpoint resource for custom MQTT broker.
+    """
+
+    helps[
+        "iot ops dataflow endpoint update eventgrid"
+    ] = """
+        type: command
+        short-summary: Update the properties of an existing dataflow endpoint resource for Azure Event Grid.
+    """
+
+    helps[
+        "iot ops dataflow endpoint update eventhub"
+    ] = """
+        type: command
+        short-summary: Update the properties of an existing dataflow endpoint resource for kafka-enabled Azure Event Hubs namespace.
+    """
+
+    helps[
+        "iot ops dataflow endpoint update fabric-onelake"
+    ] = """
+        type: command
+        short-summary: Update the properties of an existing dataflow endpoint resource for Microsoft Fabric OneLake.
+    """
+
+    helps[
+        "iot ops dataflow endpoint update fabric-realtime"
+    ] = """
+        type: command
+        short-summary: Update the properties of an existing Microsoft Fabric Real-Time Intelligence data flow endpoint.
+    """
+
+    helps[
+        "iot ops dataflow endpoint update local-mqtt"
+    ] = """
+        type: command
+        short-summary: Update the properties of an existing Azure IoT Operations Local MQTT data flow endpoint.
+    """
+
+    helps[
+        "iot ops dataflow endpoint update localstorage"
+    ] = """
+        type: command
+        short-summary: Update the properties of an existing local storage data flow endpoint.
+    """
+
+    helps[
+        "iot ops dataflow endpoint import"
```

Comment:
What is import supposed to do? I also don't see a command handler mapped to it.

Reply:
yeah this is removed, still work in progress

Reply:
had some issue with merging my old branch